### PR TITLE
Videoupload fixes

### PR DESCRIPTION
**Pull request by joschahenningsen**

### CODE REVIEW

File: web/assets/service-worker.js [hunk 1]
```diff
@@ -15,6 +16,14 @@ const PREFETCH_CACHE_FILES = [
     "/static/node_modules/@alpinejs/persist/dist/cdn.min.js",
     "/static/node_modules/alpinejs/dist/cdn.js",
 ];
+const CACHE_REQUEST_METHOD_WHITELIST = ["GET"];
+const CACHE_REQUEST_HOST_WHITELIST = [self.location.host];
```
Comment (joschahenningsen) [inline web/assets/service-worker.js]: Are you fine with `ALLOWLIST` instead? @mono424

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joschahenningsen) [inline .idea/misc.xml]: can you .gitignore this file? Somehow it always gets removed and recreated by goland :D

File: web/assets/service-worker.js (hunk 1 above)
Comment (mono424) [reply web/assets/service-worker.js]: I would say whitelist is the common term but I am basically fine with everything and `allowlist` is very understandable :)

Comment (mono424) [reply web/assets/service-worker.js]: Goland now shows ALLOWLIST as a typo, which is annoying. I still changed it for now, but if you have other proposals for the name, feel free to change it 👍

File: .idea/misc.xml (hunk 2 above)
Comment (mono424) [reply .idea/misc.xml]: yes! 👍

**Comment by joschahenningsen** [comment]
@mono424 I think this needs approval by someone other than me :D

**Review by mono424 — COMMENTED** [review]
looks good 👍

**Review by mono424 — APPROVED** [review]
👍

**Pull request merged**